wtxmgr/tx.go

```diff
@@ -47,9 +47,19 @@ type AddrTxOutput struct {
 	Index  uint32
 	Amount types.Amount
 	Block Block
-	Spend int32 // index 0  unspend 1 spend 2 coinbase
+	Spend spend
 	SpendTo *SpendTo
 }
+
+type spend int32
```

Comment:
`spend` is not a good `golang` type definition. please see https://github.com/golang/go/wiki/CodeReviewComments